config/core.entity_view_display.media.image.1_1.yml

```diff
@@ -0,0 +1,35 @@
+uuid: 65dbda2d-04c8-4645-b2e3-4b1fc23466ff
```

Comment:
In every file, we need the UUID line removed.

We have a bash script saved as a Docksal command here: https://github.com/kanopi/drupal-starter/blob/main/.docksal/commands/uuid-rm

@banoodle Wrote that.  If you have any questions on how to configure, please ping her.

Reply:
Removed UUIDs!